components/apimgt/org.wso2.carbon.apimgt.rest.api.authenticator/src/main/java/org/wso2/carbon/apimgt/rest/api/authenticator/AuthenticatorAPI.java

```diff
@@ -58,6 +60,10 @@
 import javax.ws.rs.core.NewCookie;
 import javax.ws.rs.core.Response;
 
+import static org.wso2.carbon.apimgt.rest.api.authenticator.constants.AuthenticatorConstants.HTTPS_PROTOCOL;
```

Comment:
any reason for use this kind of constant imports?

Reply:
Fixed with https://github.com/wso2/carbon-apimgt/pull/4896/commits/7f72562a3f2a9dc2595f82daf9f6b95e0fae92e9